index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portifolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="top-header">
+        <div class="icons-section">
+            <div class="time">
+                <img alt="time icon" src="images/time.png">
+            </div>
+            <div class="group-icons">
+                <img alt="icon of cellural network,wifi and battery" src="images/group.png">
+            </div>
+        </div>
+        <nav class="nav-bar">
+            <a href="#" class="logo-name">Seth Bizimana</a>
+           <a href="#" class="menu-icon">
+                <img alt="icon of menu button" src="images/Union.png">
+           </a> 
```

Comment:
I have tried to use the same properties for fonts (family, color and size) but still i doesn't look the same as in template i think it's because i do not have the fonts installed on my pc.